### ISSUE TITLE
Workflow of the complete pipeline

### ISSUE THREAD

**Issue by amarinderthind** [hartwigmedical/gridss-purple-linx#41]
Hi, I am trying to link various output from the complete pipeline. However, I find it bit hard to follow without the complete workflow diagram. E.g. only LINX plot use reportable driver files as an input. It would be great, if you can share workfow diagram of the complete pipeline. It will be helpful to qucikly understand the overall processing of gridss-purple-linx. Thanks

**Comment by charlesshale** [comment]
The Gridss-Purple-Linx project is now deprecated and replace by the following pages:

An overview of the HMF DNA tools:
https://github.com/hartwigmedical/hmftools/

A demo DNA pipeline:
https://github.com/hartwigmedical/hmftools/tree/master/pipeline
